commit 0db33a03454a0be17fbc5e849aca71d58f91849c
Author: Alexandre Belloni <alexandre.belloni@free-electrons.com>
Date:   Tue Oct 25 11:37:59 2016 +0200

    power: reset: at91-poweroff: timely shutdown LPDDR memories
    
    commit 0b0408745e7ff24757cbfd571d69026c0ddb803c upstream.
    
    LPDDR memories can only handle up to 400 uncontrolled power off. Ensure the
    proper power off sequence is used before shutting down the platform.
    
    Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
    Signed-off-by: Sebastian Reichel <sre@kernel.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1271_0db33a03454a0be17fbc5e849aca71d58f91849c.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1271_0db33a03454a0be17fbc5e849aca71d58f91849c.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
